jenkins/broadcom/buildimage-brcm-all/Jenkinsfile

```diff
@@ -33,13 +33,14 @@ pipeline {
 
 git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --relative-to=. $(cut -d" " -f2 .git))" > .git'
 
+CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=wcache SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/broadcom"
```

Comment:
>/nfs/dpkg_cache/ [](start = 70, length = 16)

Could all jobs use `/nfs/dpkg_cache/`? Is there any conflict?

Reply:
Yes, it is not a issue to put all the cache files in /nfs/dpkg_cache, it should not have any conflicts.

Use a different folder for each of the platforms, for following reasons:
1. By design, the cache cannot be used across platform.
The .platform is dependency file in all platforms. See below setting:
SONIC_COMMON_FILES_LIST :=  $(if $(wildcard cache.skip.common),, .platform slave.mk rules/functions Makefile.cache)
2. East bug fixing, it is really hard to know which platform used for the following .tgz if mixed together.
root@jenkins-worker-8:/nfs/dpkg_cache# ll vs | grep header
-rwxrwxrwx 1 johnar johnar      47499517 Mar 13 02:49 linux-headers-4.9.0-11-2-common_4.9.189-3+deb9u2_all.deb-da39a3ee5e6b4b0d3255bfe-5dfece3f63c32a62a03081b.tgz*
-rwxrwxrwx 1 johnar johnar             0 Mar 12 01:28 linux-headers-4.9.0-11-2-common_4.9.189-3+deb9u2_all.deb_cache_access.lock*
root@jenkins-worker-8:/nfs/dpkg_cache# ll broadcom/ | grep header  
-rwxrwxrwx 1 johnar johnar      47499515 Mar 13 03:45 linux-headers-4.9.0-11-2-common_4.9.189-3+deb9u2_all.deb-da39a3ee5e6b4b0d3255bfe-9dcd1a94c0b1d06751fa121.tgz*
-rwxrwxrwx 1 johnar johnar             0 Mar 12 06:22 linux-headers-4.9.0-11-2-common_4.9.189-3+deb9u2_all.deb_cache_access.lock*